docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,244 @@
+auditor no-login-6566121f-b602-47f1-a118-c9c618ee5aec             session:list,read,event:list,read                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
+editor                                                            user:list,create,read,update,delete,...
+```
+
+Machine ID can join with a token or the [IAM Method](https://goteleport.com/docs/setup/guides/joining-nodes-aws) on AWS.
+
+<Tabs>
+  <TabItem label="Token-based Joining">
+  ```code
+  $ tctl bots add robot --roles=access
+  ```
+  </TabItem>
+  <TabItem label="IAM Method">
+  First, create an IAM method token that specifies the AWS account from which
+  the bot can join.
+
+  ```
+  kind: token
+  version: v2
+  metadata:
+    # The token name is not a secret because instances must prove that they are
+    # running in your AWS account to use this token.
+    name: iam-token
+    # Set a long expiry time for how long you want to support IAM method for
+    # joining. It is safe to set this value to a very long time.
+    expires: "3000-01-01T00:00:00Z"
+  spec:
+    # Only allow bots to join using this token.
+    roles: [Bot]
+  
+    # Set the join method to be IAM.
+    join_method: iam
+
+    # Define the name of the bot that will be allowed to use this token. Note,
+    # the "bot-" prefix is required. The name of the bot you will create is "robot".
+    bot_name: bot-robot
+  
+    allow:
+    # Restrict the AWS account and (optionally) ARN that can use this token.
+    # This information can be obtained from running the
+    # "aws sts get-caller-identity" command from the CLI.
+    - aws_account: "111111111111"
+      aws_arn: "arn:aws:sts::111111111111:assumed-role/teleport-bot-role/i-*"
+  ```
+
+  Next, create the bot user.
+
+  ```
+  $ tctl bots add robot --token=iam-token --roles=access
```

Comment:
I tried IAM yesterday, and the name has to match `bot_name` set in the IAM Token YAML. Updated to match so Copy and Pasting commands works. 
```suggestion
  $ tctl bots add bot-robot --token=iam-token --roles=access
```

Reply:
This will result in a user and role named `bot-bot-robot` which might get a little awkward. I think it might be better to fix this the other way around and have `bot_name` accept, well, a bot name. Some additional discussion on this: https://github.com/gravitational/teleport/issues/10854

Reply:
@benarent I just tried the instructions as-is and they do work. Can you ping me and show me what's going on? I can help you debug. In the meantime I'll merge this PR as-is.